include/trainer_hill.h

```diff
@@ -1,7 +1,7 @@
 #ifndef GUARD_TRAINER_HILL_H
 #define GUARD_TRAINER_HILL_H
 
-extern u32 *gUnknown_0203CF5C;
+extern u32 *gVBlankCounterPointer;
```

Comment:
I'd rather have this called `gVBlankCounter`.  It's like calling something `gCounterInteger` or `MyThingStruct`.

Reply:
I renamed it `gTrainerHillVBlankCounter`.